packages/analyticsdx-template-lint/src/schemas/template-info-schema.json

```
@@ -679,6 +679,30 @@
                 "body": "${0}"
               }
             ]
+          },
+          "liveConnection": {
+            "type": "object",
+            "additionalProperties": false,
+            "required": ["connectionName", "sourceObjectName"],
+            "properties": {
+              "connectionName": {
+                "type": "string",
+                "description": "The devname of the live connection (choose something unique to avoid collisions)"
+              },
+              "sourceObjectName": {
+                "type": "string",
+                "description": "The source object name"
+              }
+            },
+            "defaultSnippets": [
+              {
+                "label": "Live Dataset",
+                "body": {
+                  "connectionName": "someuniqueliveconnectionname",
+                  "sourceObjectName": ""
```

This is how you would set it up to tab through the fields:
```suggestion
                  "connectionName": "${1:someuniqueliveconnectionname}",
                  "sourceObjectName": "${2}"
```